### ISSUE TITLE
Release Flatcar Container Linux Alpha 3794.0.0, Beta 3760.1.0 and Stable 3602.2.2

### ISSUE THREAD

**Issue by tormath1** [kinvolk/Flatcar#1258]
The release of the following Flatcar Container Linux version is planning is planning the week of 20th November.

- Alpha 3794.0.0
- Beta 3760.1.0
- Stable 3602.2.2

## 1. Preparation

Owner: @tormath1 

- [x] A. Go/No-Go [hackmd doc](https://hackmd.io/team/flatcar) created from HackMD template for test tracking and release notes: https://hackmd.io/nG2pd4iKQ9GTazucHH5U9Q
- [x] B. All planned changes are merged & blocker bugs are fixed, last nightly reviewed for problems.
- [x] C. Check that the kernel PR are merged for the channel(s) to be released.

## 2. Build

Owner: @tormath1 

- [x] A. Branched off the `main` branch (only for new major).
- [x] B. Tagged all the build branches.
- [x] C. Started the Jenkins builds, post the links into this issue as comment.

### i. The Go/No-Go Meeting

Owner: @tormath1 

- [x] A. Test status captured in the HackMD document.
- [x] B. All tests are Green.
- [x] C. Release notes / announcements added to Go/No Go doc from `container/image_changes` job by selecting `Timestamps: None`.
- [x] D. QA: Checked the diff for image file list, packages, and image size in the `container/image_changes` job output.
- [x] E. Conduct Matrix Go/No Go meeting (inc. release notes review).

### ii. Artifacts goes public

Owner: @tormath1

- [x] A. Created the Release Notes on the `scripts` repo for the respective tags on GitHub as GitHub Releases.
- [x] B. Ran the `container/release` Release job.
- [x] C. Images uploaded with `copy-to-origin.sh`.
- [x] D. Symlink to "current" updated with `set-symlink.sh`.
- [x] E. Website updated with `./update-flatcar-versions.sh` and PR merged. (https://github.com/flatcar/flatcar-website/pull/288)

### iii. Payload

Owner: @tormath1

- [x] A. Update payload signed/uploaded.
- [x] B. Release package published in Nebraska (Prerequisite: 3.ii.E should marked done).

### iv. Cloud Marketplace & Cluster API

- A. Azure: @pothos 
    - [x] Offers updated and started publishing.
    - [x] Sent the offers to publish to Live.

- B. GCP: @pothos 
    - [x] Offer deployment package uploaded to Google Bucket & submit for verification.
    - [ ] Offer Publish

- C. Cluster API for new Stable: @pothos 
  - [x] Azure: Manually run `./azure-sig.sh` to create New Shared Community Gallery Images

## 4. Announcements

Owner: @tormath1

- [x] A. Sent announcement to Mailing Lists
- [x] B. Sent Tweet out to Twitter, and Toot out to Mastodon
- [x] C. Brief version announcement in slack (k8s slack #flatcar) and the Flatcar Matrix channel

**Comment by tormath1** [comment]
Builds:
* Alpha 3794.0.0: http://jenkins.infra.kinvolk.io:8080/job/container/job/sdk/1187/cldsv/
* Beta 3760.1.0: http://jenkins.infra.kinvolk.io:8080/job/container/job/packages_all_arches/2955/cldsv/
* Stable 3602.2.2: http://jenkins.infra.kinvolk.io:8080/job/container/job/packages_all_arches/2956/cldsv/

**Comment by pothos** [comment]
In the release notes for Stable/Beta we should announce that Torcx is removed from Alpha.
Ideally we do this by linking to a blog post, see https://github.com/flatcar/Flatcar/issues/644

**Comment by pothos** [comment]
I can do Azure

**Comment by pothos** [comment]
Offers are published, need to set to live tomorrow

**Comment by pothos** [comment]
GCP offers are submitted for verification

**Comment by pothos** [comment]
> GCP offers are submitted for verification

… but publishing seems to be blocked on fulfilling a new requirement

**Comment by sayanchowdhury** [comment]
Closing in favor of #1284
